build-gluster-org/jobs/nightly-pipeline-release-template.yml

```diff
@@ -0,0 +1,30 @@
+- project:
+    name: 'nightly-release'
+    relversion:
+      - release-9
+    jobs:
+      - 'nightly-{relversion}'
+
+- job-template:
+    name: 'nightly-{relversion}'
+    description: 'Run the pipeline for branch {relversion}'
+    project-type: pipeline
+    dsl: !include-raw-escape: ../scripts/nightly-release.dsl
+
+    triggers:
+    - timed: "H 18 * * *"
+
+    publishers:
+    - email:
+        recipients: maintainers@gluster.org
+        send-to-individuals: false
+
+    parameters:
+    - string:
+        default: 'refs/heads/{relversion}'
+        description: 'For review 12345 and patch set 6, this will be refs/changes/45/12345/6. Default: Tip of master'
+        name: GERRIT_REFSPEC
```

Comment:
We have migrated from Gerrit. It would be good if we don't use such parameter name like `GERRIT_REFSPEC`. It also means that you need to edit the dsl file to pick the correct parameter values.

Reply:
I haven't changed much of the job definition I have just created a template for any more requests coming in our direction. About the above comment let me take a look and update.